packages/preferences/package.json

```diff
@@ -10,6 +10,7 @@
     "@theia/userstorage": "1.22.1",
     "@theia/workspace": "1.22.1",
     "async-mutex": "^0.3.1",
+    "p-debounce": "^2.1.0",
```

Comment:
Just surprised eslint didn't complain about this being used implicitly before?

Reply:
Right, I confirmed that it doesn't complain apparently for `require` imports (perhaps something we can update in our plugin):

https://github.com/eclipse-theia/theia/blob/1ef86ee8706f679d07c7a41e6e3d269bd55a10e3/packages/preferences/src/browser/views/preference-searchbar-widget.tsx#L20